dmutils/flask_init.py

```diff
@@ -35,6 +35,7 @@ def init_app(
     logging.init_app(application)
     proxy_fix.init_app(application)
     request_id.init_app(application)
+    force_https.init_app(application)
```

Comment:
As discussed, we'll want to override the HSTS age to something less than one year for starters.